lib/modules/manager/bundler/extract.spec.ts

```diff
@@ -267,17 +305,24 @@ describe('modules/manager/bundler/extract', () => {
     expect(res).toMatchObject({
       deps: [
         {
+          datasource: 'rubygems',
           depName: 'gem_without_values',
+          skipReason: 'unspecified-version',
         },
         {
+          currentValue: '>= 3.0.5',
+          datasource: 'rubygems',
           depName: 'gem_with_one_value',
-          currentValue: '">= 3.0.5"',
+          skipReason: 'unknown-registry',
```

Comment:
What triggered this change?

Reply:
In Gemfile, at least one registry is mandatory, but anyways I reverted back to the original Renovate as of `main` at the moment.